app/models/organisation.rb

```diff
@@ -6,6 +6,10 @@ class Organisation < ApplicationRecord
   has_many :organisation_rent_periods
   has_many :owned_schemes, class_name: "Scheme", foreign_key: "owning_organisation_id", dependent: :delete_all
   has_many :managed_schemes, class_name: "Scheme", foreign_key: "managing_organisation_id"
+  has_many :parent_organisation_relationships, foreign_key: :child_organisation_id, class_name: "OrganisationRelationship"
```

Comment:
@natdeanlewissoftwire Could you add some model tests for this? I think at a minimum we would like to test whether an organisation can have parents and children.